src/Knp/Menu/Matcher/Voter/RouteVoter.php

```diff
@@ -11,10 +11,7 @@
  */
 class RouteVoter implements VoterInterface
 {
-    /**
-     * @var RequestStack
-     */
-    private $requestStack;
+    private \Symfony\Component\HttpFoundation\RequestStack $requestStack;
```

Comment:
this should rely on the use statements

Reply:
Maybe we should add this to the php-cs-fixer rules?

Reply:
Aha, this is a pending feature request: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/5004